config.py

```diff
@@ -1,5 +1,10 @@
 
 import os
 
-chromedriver_path = os.path.dirname(os.path.realpath(__file__)) + '/resources/chromedriver.exe'
+windows_chromedriver = 'chromedriver.exe'
 
+chromedriver_path = os.path.dirname(os.path.realpath(__file__)) + '/resources/' + windows_chromedriver
+
+
+homepage_url = 'https://www.coinbase.com/'
+signInPage_url = 'https://www.coinbase.com/signin'
```

Comment:
This will be changed by the comment above, but, an additional comment, it's a bad idea to mix camel case and snake case. Pick one.